article/sections/_4_graph_construction.tex

```diff
@@ -1,3 +1,90 @@
+These post-outcome variables temporally follow the outcome variable(s) but do not cause variables in the causal graph for other observations.
+We remove all such post-outcome variables from our graph.
+
+Now is a good time to step back and consider what other researchers have thought.
+Specifically, we should conduct a literature review to see how other researchers have conceptualized the problem that we are working on.
+Have they included variables that we have not?
+Were those variables related our outcomes of interest?
+If so, should we add these variables to our causal graph, and how?
+Do the included variables of other researchers point suggest the existence of confounders in their work that we should include in our graph?
+Have other researchers ascribed differing roles to our graph's current variables than we have?
+For example, have other researchers judged a variable to be confounder, when we solely thought of the variable as an affect modifier?
+Critically examine the evidence for these alternative decisions to see if we should also reconsider how we're judging our variables.
+
+Finally, we need to connect the variables in our graph.
+\begin{enumerate}
+   \item Draw direct arrows from our explanatory variables, confounders, and effect modifiers to the outcomes.
+   \item Draw arrows from the explanatory variables to the mediators, and then draw arrows from the mediators to the outcomes.
+   \item Draw arrows from the confounders to the explanatory variables and mediators that they may cause.
+   \item Draw arrows from the variables in time $t$ to the variables that they cause in time $t+1$.
+\end{enumerate}
+After drawing in all arrows, we should now have a fully connected causal graph.
+Take a moment to look over the graph to ensure there are no remaining singletons and that we have not drawn any spurious connections.
+Also, take a moment to celebrate.
+Drawing one's first causal graph is hard work!
+
+After celebrating, take a moment to pursue the following graph editing exercises.
+First, think about how the graph might differ across sub-populations.
+What sub-populations exist in your population of interest?
+Are there any causal relationships that should not exist for a given sub-population?
+For instance, are the outcomes in some sub-populations independent of a given explanatory variable?
+Can you think of any inverted causal relationships in this sub-population?
+(I.e., for a given sub-population $B \rightarrow A$ instead of $A \rightarrow B$?)
+Consider adding these sub-population indices to one's initial causal graph, or if this is not clear enough, draw amended causal graphs for each sub-population.
+Now, one can actually relax.
+This concludes the ``purely mental'' drafting of one's causal graph.
+In the next section, we'll look at testing this graph against data, and making any edits deemed empirically necessary.
```

Comment:
```suggestion
After celebrating, take a moment to pursue the following graph editing exercises.
First, think about how the graph might differ across sub-populations.
What sub-populations, if any, exist in your population of interest?
Are there any causal relationships that should or should not not exist for a given sub-population?
For instance, are the outcomes in some sub-populations independent of a given explanatory variable?
Can you think of any inverted causal relationships specific to this sub-population?
(I.e., for a given sub-population, does $B \rightarrow A$ instead of $A \rightarrow B$?)
Consider adding these sub-population indices to one's initial causal graph, or if this is not clear enough, draw modified causal graphs for each sub-population of interest.
Now, one can actually relax.
This concludes the ``purely mental'' drafting of one's causal graph.
In the next section, we'll look at testing this graph against data, and making any edits deemed empirically necessary.
```